pkgs/development/python-modules/pixel-font-knife/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pytestCheckHook,
+  nix-update-script,
+  hatchling,
+  pypng,
+  unidata-blocks,
+  pyyaml,
+}:
+
+buildPythonPackage rec {
+  pname = "pixel-font-knife";
+  version = "0.0.7";
+  pyproject = true;
+
+  disabled = pythonOlder "3.10";
+
+  src = fetchPypi {
+    pname = "pixel_font_knife";
+    inherit version;
+    hash = "sha256-NALIf8DbIPKmazF669LHsKuP0tndR0pbphrhlXt6wyA=";
+  };
+
+  build-system = [ hatchling ];
+
+  dependencies = [
+    pypng
+    unidata-blocks
+    pyyaml
+  ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pythonImportsCheck = [ "pixel_font_knife" ];
+
+  passthru.updateScript = nix-update-script { };
```

Comment:
I think all Python modules already have an update script (based on pypi I think)

Reply:
I am not a direct user of nix, I am referring to the existing configuration.

https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/python-modules/bdffont/default.nix#L30

I'm not sure how to modify it, can you modify it directly